src/SmartComponents/Recs/List.js

```diff
@@ -39,14 +37,14 @@ const List = () => {
         <PageHeader className='ins-c-recommendations-header'>
             <PageHeaderTitle title={`${intl.formatMessage(messages.insightsHeader)} ${intl.formatMessage(messages.recommendations).toLowerCase()}`} />
             <DownloadExecReport />
-            {redhatDisabledRuleAlert && redHatDisabledRuleCount > 0 &&
+            {redhatDisabledRuleAlert === 'true' && redHatDisabledRuleCount > 0 &&
```

Comment:
```suggestion
            {redhatDisabledRuleAlert && redHatDisabledRuleCount > 0 &&
```

agree with @LI0131 makin these bools means we can simplify this line

Reply:
Done!  Yep, bools look much better than strings.